### ISSUE TITLE
Some components contain placeholder alt text

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt_framework#3328]
In [component.json](https://github.com/adaptlearning/adapt_framework/blob/6bdccc492b09fe32ca9ac9af4d8217e93b4ad8e8/src/course/en/components.json#L105), Narrative, Hotgraphic and GMCQ contain placeholder alt text. We removed the alt text from components in a previous [PR](https://github.com/adaptlearning/adapt_framework/pull/2986/files) but some got added back in a later [PR](https://github.com/adaptlearning/adapt_framework/pull/3270). I couldn't see from the ticket why these got added back in but I think it might be compatibility with the Visua11y plugin when [_noBackgroundImages](https://github.com/cgkineo/adapt-visua11y/blob/db8cfc126f8b01f9c6d72ea34aa73832addcb877/example.json#L60) is enabled. @joe-allen-89 @oliverfoster do you recall? 

My preference would be to remove all placeholder alt text as I find these often get left in courses by mistake. In most use cases, an image is decorative or content is already covered by the display text so no alt text is required. It's more of a nuisance to include alt text when not needed never mind placeholder text.

**Comment by oliverfoster** [comment]
That's just the placeholders alt text for the vanilla course. It's not going to come out in the schemas.

**Comment by kirsty-hames** [comment]
> That's just the placeholder alt text for the example course. It's not going to come out in the schemas. The vanilla course should be accessible.

Thanks @oliverfoster, are we fine to remove the placeholder alt text from the component.json then?

**Comment by oliverfoster** [comment]
No, please leave it there. It's for the default course when people from open source download the framework. It shouldn't have any impact on our projects.

**Comment by oliverfoster** [comment]
Closing
